packages/googlechrome.vm/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,7 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'Chrome'
+$category = 'Utilities'
+
+VM-Remove-Tool-Shortcut $toolName $category
```

Comment:
There is no shortcut to remove. In `chocolateyuninstall.ps1` you should uninstall GoogleChrome if it is easy to do. If it is difficult, we can remove the uninstaller.